PythonVisualizations/VisualizationApp.py

```diff
@@ -278,7 +269,7 @@ def createCodeTags(self, snippets):
     def highlightCodeTags(self, tags):
         if not isinstance(tags, (list, tuple)):
             tags = [tags]
-        for tagName in self.codeText.tag_names() if self.codeText else []:
+        for tagName in self.codeText.tag_names():
             highlight = tagName in tags
             self.codeText.tag_config(
                 tagName, 
```

Comment:
The changes to VisualizationApp.py do not belong in this PR.  If these changes go through, they would revert improvements made in other commits.